tests/tests/control_p8_lndp

```diff
@@ -64,7 +64,7 @@ export USE_MERRA2=.true.
 export IAER=1011
 export NSTF_NAME=2,1,0,0,0
 
-export LHEATSTRG=.true.
+export LHEATSTRG=.false.
```

Comment:
I think you need the DT_INNER set to something here and in the control_p8_rrtmgp test, otherwise I think you'll get an unbound variable error for these two tests.

Reply:
In default_vars.sh, dt_inner is set equal to ${DT_INNER_c96}, which is set to 360 in function export_fv3.  The problem is DT_ATMOS = 720. So I do need to set DT_INNER = ${DT_ATMOS} in control_p8_lndp.

Reply:
Yes, thanks. You are right.